edb/edgeql/compiler/setgen.py

```diff
@@ -586,6 +591,7 @@ def resolve_ptr(
     track_ref: Optional[Union[qlast.Base, Literal[False]]],
     ctx: context.ContextLevel,
 ) -> s_pointers.Pointer:
+    assert isinstance(pointer_name, str)
```

Comment:
Why this? Catching some untyped bad callsites?

Reply:
Yeah.  Not necessary anymore.